### PR TITLE
[framework] Fixed cart merging after login (closes #588)

### LINKED ISSUE #588: Transfering items from shopping cart for just logged in customer is not working
Unregistered user's items of shopping cart are not merged into new shopping cart of justlogged in customer.

### What is happening
I am using Shopsys 7.0.0.beta1

1) As an unregistered user I add few items into my shopping cart
2) I log in into my account (lets say just registered)
3) Items from my shopping cart are gone

--
4) After logging out they are back again

--

Hint: CartMigrationFacade ->mergeCurrentCartWithCart is triggered only after logout. It is not triggered after login (i am using LoginService->loginUser() method)

### PR DESCRIPTION
**Pull request by dzarda86**
| Q| A
| - | -
|Description, reason for the PR| #588 & US-4997
|New feature| No
|BC breaks| No
|Fixes issues| "closes #588"
|Standards and tests pass| Yes
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

**Comment by vitek-rostislav** [comment]
Hi, thank you for the PR :+1: 
However, it seems there was a little misunderstanding when we were discussing it - you do not need to extend the `Shopsys\FrameworkBundle\Model\Security\LoginService\LoginService` - you can directly modify it in this github repository :slightly_smiling_face:

**Comment by dzarda86** [comment]
> Hi, thank you for the PR 👍
> However, it seems there was a little misunderstanding when we were discussing it - you do not need to extend the `Shopsys\FrameworkBundle\Model\Security\LoginService` - you can directly modify it in this github repository 🙂

My bad, should be better now.

**Comment by vitek-rostislav** [comment]
Hi, as we discussed on Slack, I believe we can focus on this issue in the next sprint. We are having the sprint planning on 5th of December so I will let you know then at the latest.

File: packages/framework/src/Model/Security/LoginService.php [hunk 1]
```diff
@@ -70,5 +70,8 @@ public function loginUser(User $user, Request $request)
         // dispatch the login event
         $event = new InteractiveLoginEvent($request, $token);
         $this->traceableEventDispatcher->dispatch(SecurityEvents::INTERACTIVE_LOGIN, $event);
+
+        //migrate session
```
Comment (DavidOstrozlik) [inline packages/framework/src/Model/Security/LoginService.php]: This comment is not relevant. You can see that the session will migrate in the code and the reason why it happens should be described in the commit message (and that will be handled when this PR is merged).

Comment (grossmannmartin) [reply packages/framework/src/Model/Security/LoginService.php]: Hi @dzarda86, thank you for your time with this PR. 
I removed the comment, so we can merge and close issue and so no further action from you is required. 

We're looking forward to your contributions. ✨

**Comment by DavidOstrozlik** [comment]
TESTS: Quantity of cart items doesn't merge. If you have same product in not-logged-in and logged-in cart, it takes the quantity from the not-logged-in cart.

**Comment by grossmannmartin** [comment]
@DavidOstrozlik Thanks for the testing! I've fixed an issue and wrote proper assert, to prevent this from happening again.

**Comment by PetrHeinz** [comment]
Thanks @dzarda86! :tada:

**Pull request merged**